bitnami/sonarqube/templates/_helpers.tpl

```diff
@@ -196,3 +210,50 @@ sonarqube: database
        externalDatabase.database=DB_NAME
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create the name of the deployment's install_plugins initContainer
+*/}}
+{{- define "sonarqube.installPlugins.initContainer" -}}
+    {{ printf "%s-install-plugins-initcontainer" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create the name of the deployment's caCerts initContainer
+*/}}
+{{- define "sonarqube.caCerts.initContainer" -}}
+    {{ printf "%s-ca-certs-initcontainer" (include "common.names.fullname" .) }}
+{{- end -}}
+
```

Comment:
I see that this is being used only once. Aren't we actually creating more lines than necessary by creating a macro?